SConstruct

```diff
@@ -30,6 +30,8 @@ AddOption("--cleanbuild", dest="cleanbuild", action="store_true")
 AddOption("--pio_args", dest="pio_args", type="string", action="store", default="")
 # add option to build without debug info
 AddOption("--release", dest="release", action="store_true")
+# add option to halt openocd/gdb on startup
+AddOption("--halt", dest="halt", action="store_true")
```

Comment:
Not sure I love this. You can add this param without one of the gdb targets and it will do nothing. I think it might be better to just add a new alias in the sconscript that halts the chip